duel_dqn.py

```diff
@@ -158,6 +162,11 @@ def main(env, q, q_target, optimizer, device):
     env = gym_super_mario_bros.make("SuperMarioBros-v0")
     env = JoypadSpace(env, COMPLEX_MOVEMENT)
     env = wrap_mario(env)
+    device = 'cpu'
+    if torch.cuda.is_available():
+        device = 'cuda'
+    elif torch.backends.mps.is_available:
```

Comment:
I think this line has to be:
torch.backends.mps.is_available ->  torch.backends.mps.is_available()

Reply:
Was just trying it out and didn't really prepare to commit anything. I had to redo the fix and obviously I screw up 🤣